### DISCUSSION TITLE
Restrict horizontal movement of stencil

### DISCUSSION THREAD

**Discussion by yannskee** [advanced-cropper/vue-advanced-cropper#145] [Q&A] [answered]
Hello, first of all, great plugin! 
I would like to know if it s possible to restrict the stencil movement to vertical-only ?
Basically, I'm initially positioning the stencil in the center of the image and I'd like to only allow a vertical movement.

**Comment by Norserium** [comment, marked as answer]
@yannskee, hello! What if user zoom image by mouse wheel, for example, and move image horizontally afterwards? Will it be okay?

**Reply by yannskee** [reply]
i've deactivated the zoom

**Reply by Norserium** [reply]
What about the stencil resizing? It can be actually used to change the stencil position just like its size.

**Reply by yannskee** [reply]
stencil resizing is also disabled

**Reply by Norserium** [reply]
@yannskee, I assume that the most simple solution for your use-case is creating of the new stencil component.

**Reply by Norserium** [reply]
[There is the example](https://codesandbox.io/s/vue-advanced-cropper-vertical-stencil-ix7sk).

**Reply by yannskee** [reply]
This will do ! Thank you very much !

**Reply by Norserium** [reply]
You are welcome!
